src/main/java/uk/gov/hmcts/reform/managecase/api/payload/ApplyNoCDecisionRequest.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.managecase.api.payload;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.reform.managecase.client.datastore.CaseDetails;
+
+import javax.validation.constraints.NotNull;
+
+import static uk.gov.hmcts.reform.managecase.api.errorhandling.ValidationError.CASE_DETAILS_REQUIRED;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@ApiModel("Verify Notice of Change Answers Request")
```

Comment:
"Verify Notice of Change Answers Request" is this correct description

Reply:
Good spot 👍